django/middleware/csrf.py

```diff
@@ -14,7 +14,8 @@
 from django.utils.cache import patch_vary_headers
 from django.utils.crypto import constant_time_compare, get_random_string
 from django.utils.encoding import force_text
-from django.utils.http import same_origin
```

Comment:
It should be fine to remove as it's not documented and GitHub search didn't reveal any uses in the wild.